### ISSUE TITLE
not kicking

### ISSUE THREAD

**Issue by denizsahin7744** [Menacho15/BrandBlocker#4]
I'm using paper 1.20.6 i want to block fabric and forge clients or only allow vanilla and optifine but plugin isn't working

Config:
```
# BrandBlocker by VictorGamer15
#
# Mode types:
# Blacklist: Kick players which have the brand on the brand list below
# Whitelist: Only allow players which have the brand on the brand list below
#
# If you need some help, chat with me on the 'Discussion' tab of the plugin

# Enable the functionality of BrandBlocker
# Users with the permission brandblocker.bypass will not have any restrictions with brands on joining
enable: true

# BrandBlocker mode
mode: blacklist

# List of brands to whitelist or block
blocked-brands:
  - fabric
  - forge

# Geyser Support
geyser-support: false
geyser-prefix: "*"

# Plugin messages
prefix: "&c&lBrandBlocker &c» "
no-permission: "&7You need the permission &ebrandblocker.usage &7to use this command."
specify-player-name: "&7You need to specify a &eplayer name &7to use this command."
check-succesful: "&e%player% &7entered to the server with the client brand &e%brand%&7."
check-failed: "&7The player &e%player% &7didn't send a client brand packet, or it isn't connected."
config-reload: "&7The main configuration file was &ereloaded&7."
console-log: "%player% was kicked for using %brand%."

kick-message: |-
  &cThe client that you're using, is not permitted on our server.
  &cPlease use another client.
```

**Comment by mercurialmusic** [comment]
Confirming it's also not kicking many for me. 

I would propose instead of an internal server-specific kick attempt, allow server owners to input their desired command to run when the client is detected? I'm running a Velocity network with network-wide plugins that can manage global kicks a lot more efficiently, as I imagine most who use this plugin also have additional solutions in place.

**Comment by Menacho15** [comment]
Sorry for taking so long to fix this issue. I fixed kicking not working on 1.20.6 and now instead of having only a kick option, now you can specify the command to execute on brand detection in the configuration file. You'll need to create the config file again by deleting the old one.
